crates/re_log_types/src/data_table.rs

```diff
@@ -1231,10 +1231,16 @@ impl DataTable {
                         let c1_bytes = cell_to_bytes(c1.clone());
                         let c2_bytes = cell_to_bytes(c1.clone());
```

Comment:
This looks like it should be:
```suggestion
                        let c2_bytes = cell_to_bytes(c2.clone());
```

Reply:
good catch!